addons/skin.confluence/720p/IncludesHomeMenuItems.xml

```diff
@@ -222,7 +222,7 @@
 			<onclick>SetFocus(9000)</onclick>
 			<onclick>EjectTray()</onclick>
 		</control>
-		<control type="image" id="90126">
+		<control type="image" id="90127">
```

Comment:
I'm not a skinning expert, but imo this should read 90133 as 90127 is already used by the "system" section. At least, 90133 works fine for me. Just tried.